common/beerocks/bwl/include/bwl/mon_wlan_hal_types.h

```diff
@@ -393,19 +393,19 @@ enum eChannelScanResultChannelBandwidth : uint8_t {
 
 typedef struct sChannelScanResults {
     //The current service set identifier in use by the neighboring WiFi SSID. The value MAY be empty for hidden SSIDs.
-    char ssid[beerocks::message::WIFI_SSID_MAX_LENGTH];
+    char ssid[beerocks::message::WIFI_SSID_MAX_LENGTH] = {'\0'};
```

Comment:
```suggestion
    char ssid[beerocks::message::WIFI_SSID_MAX_LENGTH] = {0};
```

Reply:
I disagree. For me at least, `'\0'` tells me it's the zero-terminator of a C string, while `0` is just a zero that doesn't tell me much.